recipe/meta.yaml

```diff
@@ -30,35 +25,33 @@ requirements:
         - numpy x.x
         - zlib 1.2.*
         - lzo 2.06  # [linux]
-        - bzip2
+        - bzip2 1.0.x
```

Comment:
It has to be an `*` and not an `x`. (I know confusing b/c we use `*` for `numpy` :grimacing:)